src/MicrosoftTeams.ts

```diff
@@ -75,6 +75,18 @@ namespace microsoftTeams {
         Staff = 4
     }
 
+    export interface TabInstanceParameters {
+        /**
+         * Flag indicating to return all channels
```

Comment:
Nit: At least for me these flag names suggests some different from description. Maybe something like "flag allowing to select favorite teams/channels only"?

Reply:
oops.  ok :)